BaseLib/FileFinder.h

```diff
@@ -36,7 +36,8 @@ class FileFinder
     FileFinder()
     {
         addDirectory(".");
-        addDirectory(BuildInfo::source_path + "/FileIO");
+        addDirectory(BuildInfo::source_path + "/GeoLib/IO/XmlIO");
```

Comment:
I would move these setting to where calling this function (with extending this class). `FileFinder` is a general class and should not include setting for specific usage.